cmd/rpcdaemon/main.go

```diff
@@ -27,7 +28,9 @@ func main() {
 			defer borDb.Close()
 		}
 
-		apiList := commands.APIList(db, borDb, backend, txPool, mining, ff, stateCache, blockReader, agg, *cfg)
+		// TODO: Replace with correct consensus Engine
+		engine := ethash.NewFaker()
+		apiList := commands.APIList(db, borDb, backend, txPool, mining, ff, stateCache, blockReader, agg, *cfg, engine)
```

Comment:
This would need to be modified in theory, but I wasn't quite sure whether it was worth the extra work (we need extra arguments about Heimdall RPC for Polygon, etc.)